### DISCUSSION TITLE
Doc Seems wrong

### DISCUSSION THREAD

**Discussion by RudraSen2** [facebook/docusaurus#8007] [Q&A] [answered]
On the page https://docusaurus.io/docs/markdown-features/react#mdx-component-scope, it is written:

> If you want to register extra tag names (like the <highlight> tag above), you should consider [**wrapping @theme/MDXComponents**](https://docusaurus.io/docs/swizzling#wrapping), so you don't have to maintain all the existing mappings. Since the swizzle CLI doesn't allow wrapping non-component files yet, you should manually create the wrapper:

I've made the text bold which is to be considered.

How can we wrap `MDXComponents`? As I saw while running `npm run swizzle -- --list`, It shows that Wrapping MDXComponents is **Forbidden**. So I want to wrap it up. How do I do that?

**Comment by Josh-Cena** [comment, marked as answer]
It's still called wrapping, just not automated through the swizzle CLI. The text directly after that shows how you do this. In the future we may support wrapping objects.
